### PR TITLE
Prepare for v2. Update config.json and ABOUT.md. Refs #569

### PR DESCRIPTION
**Pull request by ricemery**
Prepare for v2. Update config.json and ABOUT.md. Refs #569

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "language": "Scala",
   "active": true,
-  "blurb": "",
+  "blurb": "Scala combines object-oriented and functional programming in one concise, high-level language. Scala's static types help avoid bugs in complex applications, and its JVM and JavaScript runtimes let you build high-performance systems with easy access to huge ecosystems of libraries.",
```
Comment (ErikSchierboom) [inline config.json]: I think `to huge ecosystems` should be `two huge ecosystems`

File: docs/ABOUT.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
-Scala is a object-oriented, functional language that runs on the Java Virtual Machine (JVM).  It was created by Martin Odersky.
+Scala combines object-oriented and functional programming in one concise, high-level language. Scala's static types help avoid bugs in complex applications, and its JVM and JavaScript runtimes let you build high-performance systems with easy access to huge ecosystems of libraries.
```
Comment (ErikSchierboom) [inline docs/ABOUT.md]: I think `to huge ecosystems` should be `two huge ecosystems`

File: docs/ABOUT.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
 
-Scala is a portmanteau of “Scalable Language”, which is said to mean (by the official website) that the language scales with you.
+* Features
+    * JVM Interoperability - Scala runs on the JVM, so Java and Scala stacks can be mixed for seamless integration.
+    * Type Inference - Scala is statically typed. But, Scala provides type inference to ease the burden of an advanced type system.
```
Comment (ErikSchierboom) [inline docs/ABOUT.md]: `But, Scala provides` might be rephrased to `However, Scala provides`

File: docs/ABOUT.md (hunk 2 above)
Comment (ricemery) [reply docs/ABOUT.md]: I lifted this text from the Scala web site. I think 'to' is what they had in mind. Not the number 2.

File: docs/ABOUT.md (hunk 3 above)
Comment (ricemery) [reply docs/ABOUT.md]: Agreed. This is much better. I will make this change.
Thanks

**Comment by ErikSchierboom** [comment]
Just waiting for the build to become green (I'm on my phone)

**Comment by ErikSchierboom** [comment]
Thanks!

**Comment by ricemery** [comment]
Thanks Erik!

**Pull request merged**